cpp/log/cert_test.cc

```diff
@@ -351,7 +352,7 @@ TEST_F(CertTest, TestUnsupportedAlgorithm) {
 // see anything else. Make the test fail if this is not the case to notify the
 // user that their setup is insecure.
 #ifdef OPENSSL_NO_MD2
-  EXPECT_EQ(Cert::UNSUPPORTED_ALGORITHM, legacy.IsSignedBy(legacy));
+  EXPECT_EQ(Code::UNAVAILABLE, legacy.IsSignedBy(legacy).status().error_code());
```

Comment:
For `util::Status` tests, you can use `EXPECT_THAT`, with the `StatusIs` helper (from `cpp/util/status_test_util.h`, something like this:

```C++
EXPECT_THAT(legacy.IsSignedBy(legacy).status(), StatusIs(Code::UNAVAILABLE));
```

Yeah, they flip the expected/actual with `EXPECT_THAT`. Gotta keep it exciting! :smiley: